infrastructure/cdn-in-a-box/traffic_ops/run-go.sh

```diff
@@ -142,7 +148,12 @@ mkdir -p /var/log/traffic_ops
 touch "$TO_LOG_ERROR" "$TO_LOG_WARNING" "$TO_LOG_INFO" "$TO_LOG_DEBUG" "$TO_LOG_EVENT"
 tail -qf "$TO_LOG_ERROR" "$TO_LOG_WARNING" "$TO_LOG_INFO" "$TO_LOG_DEBUG" "$TO_LOG_EVENT" &
 
-traffic_ops_golang_command=(./bin/traffic_ops_golang -cfg "$CDNCONF" -dbcfg "$DBCONF" -riakcfg "$RIAKCONF");
+if [[ -z $TV_BACKEND ]]; then
+  traffic_ops_golang_command=(./bin/traffic_ops_golang -cfg "$CDNCONF" -dbcfg "$DBCONF" -riakcfg "$RIAKCONF");
```

Comment:
so if both `-riakcfg` and `traffic_vault_backend` (in cdn.conf) are used, I believe `traffic_vault_backend` will take precedence. So we would also have to exclude the `traffic_vault_backend`-related config in cdn.conf in order to use this line to enable Riak. Or, we can actually specify the riak config in `traffic_vault_backend` in cdn.conf (and omit the `riakcfg`) if we would like to use Riak instead of postgres.

Reply:
Nevermind, we chatted about this offline, and now I understand why this works the way it is. 😄